src/test/java/io/cdap/delta/bigquery/BigQueryTargetTest.java

```diff
@@ -193,7 +193,7 @@ public void testGetMaximumExistingSequenceNumberForRetryableFailures() throws Ex
         bqTarget.initialize(deltaTargetContext);
       } finally {
         //verify at least 1 retry happens
-        PowerMockito.verifyStatic(BigQueryUtils.class, Mockito.atLeast(2));
+        PowerMockito.verifyStatic(BigQueryUtils.class, Mockito.atLeast(1));
```

Comment:
why did this test change? It seems unrelated to your changes

Reply:
As per the new code changes, we do need to mock it twice, as it only requires 1 retry and if  mocked twice. it break test in the second retry.